### PR TITLE
feat(active-release) cleanup feature flags for active release notifications

### PR DESCRIPTION
**Pull request by snigdhas**
Cleanup feature flags for release notifications:

* Remove `organizations:active-release-notification-opt-in`, removing per-user access.
* Replace `projects:active-release-monitor-default-on` with `organizations:active-release-notifications-enable`

With this change, `organizations:active-release-notifications-enable` is main flag to add organizations for release notifications. `organizations:active-release-monitor-alpha` is still around and should be the only other flag related to release notifications.

Fixes WOR-2072

### CODE REVIEW

File: src/sentry/features/__init__.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
 # Organization scoped features that are in development or in customer trials.
 default_manager.add("organizations:active-release-monitor-alpha", OrganizationFeature, True)
-default_manager.add("organizations:active-release-notification-opt-in", OrganizationFeature, True)
+default_manager.add("organizations:active-release-notifications-enable", OrganizationFeature, True)
```
[the remark this answers is not in the document]
Comment (snigdhas) [reply src/sentry/features/__init__.py]: Ah good point, removed the parameter

**Review by armenzg — COMMENTED** [review]
I don't know enough to set an approval.

File: src/sentry/features/__init__.py [hunk 2]
```diff
@@ -58,7 +58,7 @@
 
 # Organization scoped features that are in development or in customer trials.
 default_manager.add("organizations:active-release-monitor-alpha", OrganizationFeature, True)
-default_manager.add("organizations:active-release-notification-opt-in", OrganizationFeature, True)
+default_manager.add("organizations:active-release-notifications-enable", OrganizationFeature)
```
[the remark this answers is not in the document]
Comment (barkbarkimashark) [reply src/sentry/features/__init__.py:61]: I believe if that param is set to `True`, that will call out to flagr. In this case, we don't wanna do that and will be delegating to the hard-coded handler in `getsentry`.

**Review by barkbarkimashark — APPROVED** [review]

**Comment by barkbarkimashark** [comment]
Probably need to wait until [this](https://github.com/getsentry/sentry/pull/38063) gets re-reverted and fixed, then rebase your branch after.

File: src/sentry/conf/server.py [hunk 3]
```diff
@@ -948,8 +948,8 @@ def create_partitioned_queues(name):
     "auth:register": True,
     # Workflow 2.0 Alpha Functionality for sentry users only
     "organizations:active-release-monitor-alpha": False,
-    # Workflow 2.0 Experimental ReleaseMembers who opt-in to get notified as a release committer
-    "organizations:active-release-notification-opt-in": False,
+    # Workflow 2.0 Active Release Notifications
+    "organizations:active-release-notifications-enable": False,
```
[the remark this answers is not in the document]
Comment (snigdhas) [reply src/sentry/conf/server.py:952]: It's controlled through the handler here #8129

**Pull request merged**